package/MDAnalysis/analysis/nuclinfo.py

```diff
@@ -763,3 +763,54 @@ def pseudo_dihe_baseflip(universe, bp1, bp2, i,
     pseudo = mdamath.dihedral(x[0] - x[1], x[1] - x[2], x[2] - x[3])
     pseudo = np.rad2deg(pseudo) % 360
     return pseudo
+
+def angle_between_base_planes(universe, b1, b2, seg1="SYSTEM", seg2="SYSTEM"):
+    """The angle between the planes for given two bases is computed. 
+
+    .. Note:: This angle calculation will only work if using atom names as
+              documented by charmm force field parameters.
+
+    Parameters
+    ----------
+    universe : Universe
+        :class:`~MDAnalysis.core.universe.Universe` containing the
+        trajectory
+    b1 : int
+        resid of the first base
+    b2 : int
+        resid of the second base
+    segid1 : str (optional)
+        segid of b1
+    segid2 : str (optional)
+        segid of b2
+
+    Returns
+    -------
+    float
+          angle in degrees
+
+
+    .. versionadded:: 1.0.1
+    """
+    baseatoms = { 'CYT': ['N1', 'N3', 'C5'],
```

Comment:
I'm not sure I agree with the premise of having hardcoded atom names and residues for things like this. It is always best for things like this to leave the selection to users and instead ask that folks pass in either directly coordinates or atomgroups.

Reply:
Admittedly, this follows the style of the (very old) nuclinfo module. It would be consistent with the legacy code that's already there.